eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -1504,6 +1511,15 @@
     <Bug pattern="DLS_DEAD_LOCAL_STORE"/>
   </Match>
 
+  <Match>
+    <Or>
+      <Class name="com.microsoft.azure.messaging.servicebus.perf.ReceiveAndDeleteMessageTest"/>
```

Comment:
Is there a reason we have this suppression rather than removing the local variable? Our goal is to limit spotbug excludes as much as possible.